srcts/src/shiny/shinyapp.ts

```diff
@@ -253,6 +258,15 @@ class ShinyApp {
     return socket;
   }
 
+  async startInMessageQueueLoop(): Promise<void> {
```

Comment:
Should wrap the content of this function in a `try`-`catch` and print any errors to the console.